lima-init.sh

```diff
@@ -49,15 +49,17 @@ awk -f- "${LIMA_CIDATA_MNT}"/user-data <<'EOF' >> /etc/fstab
     flag = 0
 }
 flag {
+    # Use a pattern unlikely to appear in a filename. "\0" unfortunately doesn't work.
+    FS = "<;><><;>"
     sub(/^ *- \[/, "")
     sub(/"?\] *$/, "")
-    gsub("\"?, \"?", "\t")
-    print $0
+    gsub("\"?, \"?", FS)
```

Comment:
And this… replaces `", "` with the field separator? So it's a very limited YAML parse.